package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "npm-blue-tape",
+  "description": "The type definition for https://github.com/spion/blue-tape",
+  "version": "0.0.1",
+  "author": "blakeembrey",
```

Comment:
Last thing, should I keep `typescript` as `devDeps` or move it to `peerDeps`?
I am locking the version locally for vscode by the `settings.json`:
```js
{
  "typescript.tsdk": "node_modules/typescript/lib"
}
```
As now `1.8.7` is released, should I assume it would be ok to use the global instance instead?